CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.4.1
+
+Release date: TBD
+
+### Bug Fixes
+
+#### macOS
```

Comment:
I see, that makes sense.

Propose we use the same naming convention for 3.4.1 as for 3.4, and we can consider changing it for the next desktop release.